src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -756,6 +753,28 @@ private void validateRules() {
           winnerMode.toString());
     }
 
+    // nonIntegerWinningThreshold and hareQuota are only allowed for multi-seat elections
+    if (!isMultiSeatAllowOnlyOneWinnerPerRoundEnabled()
+        && !isMultiSeatAllowMultipleWinnersPerRoundEnabled()) {
```

Comment:
perhaps it would be useful to have a helper like `isStandardMultiSeatEnabled()` that matches these two modes?

Reply:
If we end up doing the same check in the future, I'd be on board with this, but will leave as-is for now.